cluster/network.go

```diff
@@ -485,26 +498,50 @@ func (c *Cluster) doWeaveDeploy(ctx context.Context, data map[string]interface{}
 }
 
 func (c *Cluster) doAciDeploy(ctx context.Context, data map[string]interface{}) error {
-	_, clusterCIDR, err := net.ParseCIDR(c.ClusterCIDR)
-	if err != nil {
-		return err
+	var podIPPool []IPPool
+	var podNetwork []PodIPNetwork
+	var podSubnet []string
+
+	clusterCIDRs := strings.Split(c.ClusterCIDR, ",")
+	for _, clusterCIDR := range clusterCIDRs {
+		podSubnet = append(podSubnet, fmt.Sprintf("\"%s\"", clusterCIDR))
+		_, ClusterCIDRs, err := net.ParseCIDR(clusterCIDR)
+		if err != nil {
+			return err
+		}
+		podIPStart, podIPEnd := cidr.AddressRange(ClusterCIDRs)
+		podIPPool = append(podIPPool, IPPool{Start: cidr.Inc(cidr.Inc(podIPStart)), End: cidr.Dec(podIPEnd)})
+		podNetwork = append(podNetwork, PodIPNetwork{Subnet: *ClusterCIDRs, Gateway: cidr.Inc(podIPStart)})
 	}
-	podIPStart, podIPEnd := cidr.AddressRange(clusterCIDR)
-	_, staticExternalSubnet, err := net.ParseCIDR(c.Network.Options[AciStaticExternalSubnet])
-	if err != nil {
-		return err
+
+	var staticServiceIPPool []IPPool
+	staticExternalSubnets := strings.Split(c.Network.AciNetworkProvider.StaticExternalSubnet[0], ",")
```

Comment:
Couple of questions/concerns.

1. Can the `c.Network.AciNetworkProvider.StaticExternalSubnet` be guaranteed non-empty? I see a check for `nil` but it would allow an empty slice.
2. It doesn't look like we access any elements past zero-index one. What's the point of it being a slice then? (And we split the only element on commas to get an array to boot)